rh-apicast-operator/Containerfile.apicast-operator-bundle

```diff
@@ -26,16 +26,18 @@ LABEL operators.operatorframework.io.metrics.project_layout=go.kubebuilder.io/v3
 LABEL operators.operatorframework.io.test.mediatype.v1=scorecard+v1
 LABEL operators.operatorframework.io.test.config.v1=tests/scorecard/
 
-LABEL com.redhat.component="3scale-apicast-operator-bundle-container" \
-      name="3scale-amp2/apicast-rhel7-operator-metadata" \
+LABEL name="3scale-amp2/apicast-rhel7-operator-metadata" \
```

Comment:
@jjkiely do these make sense to you?
The plan is that this will be alpha stream